common/static/coffee/src/discussion/discussion_module_view.coffee

```diff
@@ -97,7 +97,7 @@ if Backbone?
       else
         @$el.append($discussion)
 
-      @newPostForm = $('.new-post-article')
+      @newPostForm = this.$el.find('.new-post-article')
```

Comment:
When we were creating the newPostForm form $ instead of its own $el it includes both of new-post-article in their control which cause problem when we use the different button actions.